libs/barista-components/stacked-series-chart/src/stacked-series-chart.scss

```diff
@@ -81,11 +81,19 @@ See stacked-series-chart.layout.md
   }
 }
 
+.dt-stacked-series-chart-track-label {
+  text-align: center;
+}
+
 /** Legend */
 .dt-stacked-series-chart-legend {
   display: flex;
   padding: 0;
   justify-self: center;
+
+  &.dt-stacked-series-legend-compact-mode {
+    margin-top: 70px;
```

Comment:
I wonder if this is the best approach. We're affecting the x-axis so my initial idea would be to add this margin to the value-axis. I know the result would be the same but conceptually I think it's more logical. And one step beyond, why not add a height to `.dt-stacked-series-chart-track-label-compact-mode` ?

Reply:
Makes sense!